README.md

```diff
@@ -32,9 +32,11 @@ It is also used to scaffold contig pairs linked by ARCS/ARKS.
 
 ### Compilation:
 -------------------------------
-If cloning directly from the repository run:
+If compiling directly from within the repository you need also bundled copy of ./btllib so that `"btllib/bloom_filter.hpp"` can be found during compilation:
```

Comment:
Yes you are right `LINKS` depends on `btllib` repository, but not only for `"btllib/bloom_filter.hpp"` but for few more scripts. I believe they shouldn't be indicated in README for the sake of readability.

Reply:
Please instruct user to fetch `btlib` from some URL, unpack it, install it, and just depend on it and test for it in the `configure` script. Every sysadmin is used to that, it is clean and the only way to go. It is much more work to unbundle the bundled copies of some libs, sometimes they are even modified variants of their original, which is a nightmare.

No, definitely it should be documented in the README. Why should I even study what the thing requires for its runtime? The `make install` left me with a broken, partial installation and there is not a single explanation what's up. after poking around one only realizes one is supposed to run the code from the unpacked tree and the unfinished `make install` target is just confusing. Thank you.

Reply:
Nobody acted upon the issue https://github.com/bcgsc/LINKS/issues/31 hence my pull request.